addons/web/static/src/core/install_prompt/install_prompt.js

```diff
@@ -0,0 +1,24 @@
+/** @odoo-module **/
+
+import { Component } from "@odoo/owl";
+import { Dialog } from "@web/core/dialog/dialog";
+
+export class InstallPrompt extends Component {
+    static props = {
+        close: true,
+        onClose: { type: Function },
+        isMobileSafari: { type: Boolean },
+    };
+    static components = {
+        Dialog,
+    };
+    static template = "web.InstallPrompt";
+
+    get sizeClass() {
+        return this.props.isMobileSafari && this.env.isSmall ? "mobile" : "desktop";
+    }
```

Comment:
After re-reading it... Do we really need this ? Could usual media-queries being enough as the `InstallPrompt` is only triggered on Apple's OS (either macOS or iOS)

Reply:
Yes, why not :)